.cbmc-batch/jobs/aws_cryptosdk_md_abort/Makefile

```diff
@@ -35,6 +36,11 @@ DEPENDENCIES += $(SRCDIR)/helper-src/openssl/bn_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/ec_override.goto
 DEPENDENCIES += $(SRCDIR)/helper-src/openssl/evp_override.goto
 
+REMOVE_FUNCTION_BODIES += --remove-function-body EVP_MD_CTX_free
+REMOVE_FUNCTION_BODIES += --remove-function-body evp_md_ctx_is_valid
```

Comment:
If this is due to runtime/memory issues with CBMC, just add a comment explaining why these have to be abstracted to the proofs. Otherwise, looks good to me!

Reply:
Thanks! I've updated this as per https://github.com/aws/aws-encryption-sdk-c/pull/582#discussion_r478660427 and added some documentation to the Makefiles about why we use the abstractions.

Reply:
Verified that the issue raised by tawdry-audrey is resolved. Please update the branch and an admin can override the change request blocking merge.